flow/util/autotuner/flowautotuner_hyperopt.py

```diff
@@ -22,10 +22,10 @@
 
 
 ## User-defined evaluation function
+a = -10000
+b = 1
+c = 1000
 def evaluation_fn(step, a, ws, b, wl, c, ndrc):
```

Comment:
These are arguments and global both?  It's confusing.  Also the names could be more descriptive.

Reply:
Hi Matt, I applied all your comments to the new commits. Thank you so much for your comments!